src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -633,6 +633,22 @@
     </Environment>
   </Provider>
 
+  <Provider Name="NextCloud" Id="f256847d-c720-4b2f-bda0-69e8f4bceac2"
```

Comment:
AFAICT by looking at the official website and the wiki page, the correct casing is `Nextcloud`. Can you please change it?

Reply:
Done